azure-mgmt-storage/src/main/java/com/microsoft/azure/management/storage/implementation/StorageAccountsInner.java

```diff
@@ -836,7 +847,7 @@ public StorageAccountInner call(ServiceResponse<StorageAccountInner> response) {
                 @Override
                 public Observable<ServiceResponse<Page<StorageAccountInner>>> call(Response<ResponseBody> response) {
                     try {
-                        ServiceResponse<PageImpl<StorageAccountInner>> result = listDelegate(response);
+                        ServiceResponse<PageImpl1<StorageAccountInner>> result = listDelegate(response);
```

Comment:
Now the operation of listing storage accounts is using PageImpl1. And PageImpl will only work for those don't have next link.